frontend/src/libraries/explorer-wamp/contracts.ts

```diff
@@ -56,8 +56,12 @@ export default class ContractsApi extends ExplorerApi {
     };
 
     try {
-      const codeHash = await this.queryCodeHash(id);
-      if (codeHash !== "11111111111111111111111111111111") {
+      // codeHash precheck is requered due to deleted accounts
+      let codeHash;
+      await this.queryCodeHash(id)
+        .then((it) => (codeHash = it))
+        .catch(() => {});
```

Comment:
This should work, I believe. Please, double-check

```suggestion
      // codeHash does not exist for deleted accounts
      const codeHash = await this.queryCodeHash(id).catch(() => null);
```

Reply:
Yes, you were right!